src/main/java/org/terasology/launcher/gui/javafx/SettingsController.java

```diff
@@ -116,9 +112,9 @@
     @FXML
     private Button cancelSettingsButton;
     @FXML
-    private ComboBox<JobItem> jobBox;
+    private ComboBox<ApplicationController.PackageItem> jobBox;
```

Comment:
The reference to `ApplicationController.PackageItem` indicates that `PackageItem` should live somewhere else. I think cross references to inner classes are often a bad smell.

Reply:
Completely agreed! Tbh this was just a tiny hack to get things working quick :sweat_smile: